src/scripts/eos-plot

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
You told me that python3 is not available on fedora because `python` is python 3.x and 2.x is not available any more. Would the code break on fedora?

Reply:
Sorry, I meant CentOS, see above. I do not know of anybody getting EOS to build with python support on CentOS yet. Still trying myself.